backend/migrations/002_fix_duplicate_emails.py

```
@@ -0,0 +1,271 @@
+            f"  ✓ Keeping: ID {keep_user['id']} ({keep_user.get('name', 'Unknown')})"
+        )
+
+        for user in delete_users:
+            user_id = user["id"]
+            logger.info(f"  ✗ Deleting: ID {user_id} ({user.get('name', 'Unknown')})")
+
+            # Delete the user
+            result = await db.users.delete_one({"_id": user_id})
+            if result.deleted_count > 0:
+                total_deleted += 1
+                logger.info(f"    → Deleted successfully")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)